test/browser/spec/ol/renderer/Map.test.js

```diff
@@ -507,82 +507,121 @@ describe('ol/renderer/Map.js', function () {
-      }));
-      expect(hit).to.be.ok();
-      expect(hit.feature).to.be(feature);
-      expect(hit.layer).to.be(layer);
-      expect(hit.geometry).to.be(geometry);
 
-      layer.setStyle({
-        'text-value': 'X',
-        'text-font': 'bold 100px sans-serif',
-        'text-baseline': 'top',
-        'text-offset-y': -50,
-        'text-stroke-width': 1,
-        'text-stroke-color': 'black',
-        'text-fill-color': 'none',
-        'text-placement': 'line',
-        'text-overflow': true,
+      it('with no fill', function (done) {
+        layer.setStyle({
+          'text-value': 'X',
+          'text-font': 'bold 100px sans-serif',
+          'text-baseline': 'top',
+          'text-offset-y': -50,
+          'text-stroke-width': 20,
+          'text-stroke-color': 'black',
+          'text-fill-color': 'none',
+          'text-placement': 'line',
+          'text-overflow': true,
+        });
+        map.once('rendercomplete', () => {
+          try {
+            const hit = map.forEachFeatureAtPixel(
+              [50, 50],
+              (feature, layer, geometry) => ({
+                feature,
+                layer,
+                geometry,
+              })
+            );
+            expect(hit).to.be.ok();
+            expect(hit.feature).to.be(feature);
+            expect(hit.layer).to.be(layer);
+            expect(hit.geometry).to.be(geometry);
+            done();
+          } catch (e) {
+            done(e);
+          }
+        });
       });
-      map.renderSync();
-      hit = map.forEachFeatureAtPixel([50, 50], (feature, layer, geometry) => ({
-        feature,
-        layer,
-        geometry,
-      }));
-      expect(hit).to.be(undefined);
 
-      layer.setStyle({
-        'text-value': 'X',
-        'text-font': 'bold 100px sans-serif',
-        'text-baseline': 'top',
-        'text-offset-y': -50,
-        'text-stroke-width': 1,
-        'text-stroke-color': 'black',
-        'text-fill-color': 'transparent',
-        'text-placement': 'line',
-        'text-overflow': true,
+      it('with default fill', function (done) {
+        layer.setStyle({
+          'text-value': 'X',
+          'text-font': 'bold 100px sans-serif',
+          'text-baseline': 'top',
+          'text-offset-y': -50,
+          'text-stroke-width': 1,
+          'text-stroke-color': 'black',
+          'text-fill-color': 'none',
+          'text-placement': 'line',
+          'text-overflow': true,
+        });
+        map.once('rendercomplete', () => {
+          try {
+            const hit = map.forEachFeatureAtPixel(
+              [50, 50],
+              (feature, layer, geometry) => ({
+                feature,
+                layer,
+                geometry,
+              })
+            );
+            expect(hit).to.be(undefined);
+            done();
+          } catch (e) {
+            done(e);
+          }
+        });
+      });
+
+      it('with transparent fill', function (done) {
```

Comment:
@mike-000 This test is failing on MacOS, and to this point I'm unable to find out why.